common/utils/string.ts

```diff
@@ -1,3 +1,7 @@
 export function toHtmlId(s: string): string {
   return s.replace(/\W/g, '-');
 }
+
+export function removeHtmlTags(s: string): string {
+  return s.replace(/<[^>]+>?/gm, '');
```

Comment:
I'm afraid this won't cut it – a bunch of titles use angle brackets to indicate dates, e.g. https://wellcomecollection.org/works/yufr6m57 This function would remove those dates, even though they're a legitimate part of the title.

You don't need to strip all HTML here, because the API is already doing most of it for you – if you look at the titles in a [catalogue snapshot](https://developers.wellcomecollection.org/datasets), you see the only HTML tags we see with any frequency are `<i>` and `<I>`.

I'd suggest changing this:

* Hard-code replacements for the HTML tags we actually care about (`<i>` and `<I>`)
* Add some tests for this function, so